### PR TITLE
jaxec instead of VerboseProcess

### PR DESCRIPTION
**Pull request by yegor256**
Jaxec library is now used instead of `VerboseProcess` from jcabi-log, which helps fix the bug of #2457. Without Jaxec it would harder to fix it, since it would require some code duplication in `XMIRTest`.

---

## PR-Codex overview
This PR focuses on updating dependencies and adding a new dependency for Jaxec. 

### Detailed summary
- Updated Maven dependencies in multiple modules
- Added Jaxec dependency in multiple modules
- Updated Sonar and daily workflows to include TeXLive setup
- Updated log4j.properties file
- Updated .gitignore file
- Updated XMIRTest and MainTest classes to use Jaxec instead of VerboseProcess
- Updated BinarizeMojo and SnippetTestCase classes to use Jaxec instead of VerboseProcess

> The following files were skipped due to too many changes: `eo-maven-plugin/src/test/java/org/eolang/maven/it/SnippetTestCase.java`

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

**Comment by yegor256** [comment]
@volodya-lombrozo please, check

**Comment by yegor256** [comment]
@volodya-lombrozo ping

File: eo-parser/src/test/java/org/eolang/parser/XMIRTest.java [hunk 1]
```diff
@@ -144,6 +130,20 @@ void convertsAntlrToEbnf() throws Exception {
...
             )
         );
+        Files.write(
+            temp.resolve("article.tex"),
+            new TextOf(new ResourceOf("tex/ebnf.tex"))
+                .asString()
+                .replace("EBNF", ebnf)
+                .getBytes(StandardCharsets.UTF_8)
+        );
+        new Jaxec(
```
Comment (volodya-lombrozo) [inline eo-parser/src/test/java/org/eolang/parser/XMIRTest.java:140]: @yegor256 What do you think if we test this in a separate unit test? It looks like a clear example of `Cuckoo` anti-pattern:
https://www.yegor256.com/2018/12/11/unit-testing-anti-patterns.html

**Review by volodya-lombrozo — CHANGES_REQUESTED** [review]
@yegor256 Could you please assign your PR to an issue which you are trying to solve?
For example, you can add one of the [keywords](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword) followed by issue number.

**Comment by volodya-lombrozo** [comment]
@yegor256 What about other comments?

**Comment by yegor256** [comment]
@volodya-lombrozo updated and answered, see above

**Review by volodya-lombrozo — CHANGES_REQUESTED** [review]
@yegor256 I've finally read the entire method that you are trying to extend. I noticed, based on the method's comment, that it might not be best placed here. It's a bit puzzling how this method found its way here, but that's okay. 
While reviewing, I observed that this method aligns with a few anti-patterns, such as:
* Giant
* Free Ride
* Excessive Setup
* (Maybe more)
https://www.yegor256.com/2018/12/11/unit-testing-anti-patterns.html

How about we consider adding a task or a puzzle to possibly refactor this test? Or, if it makes sense, we might think of replacing it with a more suitable CI pipeline. 
___
Also I just wanted to point out something regarding the issue title of this PR - "jaxec instead of VerboseProcess." From what I observed, we still have instances of `VerboseProcess` in both `BinarizeMojo` and `SnippetTestCase`.
Might I suggest that we consider adding a task or a puzzle to address replacing those instances as well? Just to ensure consistency throughout our codebase.

**Review by volodya-lombrozo — APPROVED** [review]
@yegor256 Looks good to me, thank you. 
I created the separate [issue](https://github.com/objectionary/eo/issues/2464) to solve the problem with `XMIRTest#convertsAntlrToEbnf()` method.

**Comment by yegor256** [comment]
@rultor merge

**Pull request merged**